### PR TITLE
feat(webapp): add chinese translation

### LINKED ISSUE #68: Chinese Translation

### PR DESCRIPTION
**Pull request by xavier506**
### What does this PR do?

- Resolves #68

### CODE REVIEW

File: webapp/src/language/zh.json [hunk 1]
```diff
@@ -1,79 +1,77 @@
...
+  "bPs": "BPs",
+  "about": "关于",
+  "exchanges": "交易所",
+  "eosHolder": "持有者",
+  "htmlTitle": "MyvoteEOS 代理",
+  "homeTitle": "欢迎来到MyvoteEOS 代理",
+  "newsTitle": "EOS 新闻",
+  "logout": "登出",
+  "joinCommunity": "加入社区",
+  "copyRight": "拥有你的选票",
```
Comment (leisterfrancisco) [inline webapp/src/language/zh.json:15]: Using Google Translator as a support tool, this copyRight seems to be incorrect.

The word should be `版权`

File: webapp/src/language/zh.json [hunk 2]
```diff
@@ -1,79 +1,77 @@
...
+  "keepBps": "让Bps负责",
+  "legitimacy": "合法性",
+  "accountability": "问责制",
+  "interestAlignment": "兴趣调整",
+  "selectionCriteria": "投票原则",
+  "eligibility": "只有那些通过签署以下承诺加入#MyvoteEOS联盟的bp才将被考虑投票：",
+  "joinTitle": "联盟中的BPs",
+  "exchangeStatus": "交易所状态",
+  "twitToUrge": "Tweet通知他们",
+  "participating": "参与eos会员",
```
Comment (leisterfrancisco) [inline webapp/src/language/zh.json:53]: Shouldn't be it `EOS`

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: webapp/src/language/zh.json (hunk 1 above)
Comment (xavier506) [reply webapp/src/language/zh.json:15]: I got this translation fr0om hahn

File: webapp/src/language/zh.json (hunk 2 above)
Comment (xavier506) [reply webapp/src/language/zh.json:53]: probably

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**